scripts/promote_staging_to_prod.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+checks=$(curl -s  "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage") | .conclusion' | head -n 1)
+if [ "$checks" == "success" ]; then
+  echo "All checks passed in most recent staging deployment!"
+else
+  echo "Some checks failed in most recent staging deployment. Please check GHA runs."
+  exit 1
+fi
+
+echo "fetch origin branch history"
+git fetch
+echo "Stash any local changes"
+git stash
+echo "Checking out 'staging' branch and pull latest"
+git checkout staging
+git pull origin staging
+echo "Checking out 'prod' branch and pull latest"
+git checkout prod
+git pull origin prod
```

Comment:
We often have hiccups on staging and prod branches due to hot fixes and (accidental) merge commits -- a `pull` might fail for either of these branches. I think it might be better to delete the staging/prod branch and create anew from origin after a `fetch`.